web-ui/src/components/team-member/TeamMemberSelect.test.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import TeamMemberSelect from "./TeamMemberSelect";
+import { render, fireEvent, screen } from "@testing-library/react";
+
+let teamMembers;
+
+const handleMemberSelect = (members) => {
+  console.log("Parent Comp", members);
+};
+
```

Comment:
should there be a console.log here?

Reply:
Yeah just so I can use the members arg when the test runs